test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -362,7 +364,47 @@ static public void testWithFieldOnNonExistentClass() throws Throwable {
 			Assert.fail("should throw error. Class does not exist");
 		} catch (NoClassDefFoundError e) {}
 	}
-	
+
+	/*
+	 * Test ifacmp on value class
+	 * 
+	 * class TestIfacmpOnValueClass {
+	 *  
+	 * }
+	 */
+@Test(priority=1)
+  	static public void TestIfacmpOnValueClass() throws Throwable {
+    		String fields[] = {"IntField:I"};
+    		Class<?> acmpClass = ValueTypeGenerator.generateValueClass("acmpClass", fields);
```

Comment:
You dont need to make a new class, you can use one of the existing ones, like Point2D